assets/src/components/nav/leftNav.tsx

```diff
@@ -112,6 +112,9 @@ const LeftNav = ({
               }
               title={mapMode.title}
               to={mapMode.path}
+              onClick={() => {
+                mapMode.navEventText && window.FS?.event(mapMode.navEventText)
```

Comment:
nit(non-blocking): this seems like it's not part of the requirements product asked for, but I think having dynamic variables be structured event data instead of the actual message is a more stable interface.

placeholder event text and key are just examples, I'm not settled on those as final, but only for demonstration.

```suggestion
                mapMode.navEventText && window.FS?.event("clicked navigation tab", { mode: mapMode.navEventText })
```